skunkworks_crow/src/main/java/org/odk/share/tasks/UploadJob.java

```diff
@@ -384,14 +392,14 @@ private void sendBlankForm(Cursor cursor) {
                     dos.writeInt(0);
                 }
 
-                sbResult.append(displayName + " ");
-                if (formVersion != null) {
-                    sbResult.append(getContext().getString(R.string.version, formVersion));
-                }
-                sbResult.append(getContext().getString(R.string.id, formId) + " " +
-                        getContext().getString(R.string.success, getContext().getString(R.string.blank_form_count,
-                                getContext().getString(R.string.sent))));
+                sbResult.append(getContext().getString(R.string.form_transfer_result,
+                        getContext().getString(R.string.success, ", " +
+                                getContext().getString(R.string.blank_form_count,
+                                        getContext().getString(R.string.received)))));
```

Comment:
Its a `UploadJob` so `R.string.sent` should be there instead of `R.string.received`

Reply:
Oops, thanks for reminding, will fix soon.